Source/cg_SourceSnapshot.hpp

```diff
@@ -56,6 +56,15 @@ struct SourcesSnapshots {
     SourceSnapshot & operator[](SourceIndex const index);
     int size() const { return secondaries.size() + 1; }
 
+    juce::String toString ()
+    {
+        juce::String ret;
+        ret += primary.position.toString() + ", " + primary.z.toString() + "; ";
```

Comment:
hopefully one day juce supports std::format :D with it it would be: 

    ret += std::format("{}, {}; ", primary.position, primary.z);